### ISSUE TITLE
Error getting metrics

### ISSUE THREAD

**Issue by lypht** [robscott/kube-capacity#1]
Taking kube-capacity for a spin, and error with:
```joshhull at Joshs-MBP in ~
$ kube-capacity
Error getting metrics
panic: the server could not find the requested resource (get nodes.metrics.k8s.io)

goroutine 1 [running]:
github.com/robscott/kube-capacity/pkg/capacity.getMetrics(0xc000116700, 0xc000371b90)
	/Users/rob/go/src/github.com/robscott/kube-capacity/pkg/capacity/list.go:66 +0x30a
github.com/robscott/kube-capacity/pkg/capacity.List(0x28b4dd0, 0x0, 0x0, 0x0)
	/Users/rob/go/src/github.com/robscott/kube-capacity/pkg/capacity/list.go:29 +0x3e
github.com/robscott/kube-capacity/pkg/cmd.glob..func1(0x288d420, 0x28b4dd0, 0x0, 0x0)
	/Users/rob/go/src/github.com/robscott/kube-capacity/pkg/cmd/root.go:38 +0xe1
github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra.(*Command).execute(0x288d420, 0xc0000381b0, 0x0, 0x0, 0x288d420, 0xc0000381b0)
	/Users/rob/go/src/github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x288d420, 0x288d680, 0xc000133f50, 0x1b6061e)
	/Users/rob/go/src/github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra.(*Command).Execute(0x288d420, 0x10053b0, 0xc00009c058)
	/Users/rob/go/src/github.com/robscott/kube-capacity/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/robscott/kube-capacity/pkg/cmd.Execute()
	/Users/rob/go/src/github.com/robscott/kube-capacity/pkg/cmd/root.go:49 +0x2d
main.main()
	/Users/rob/go/src/github.com/robscott/kube-capacity/main.go:22 +0x20
```

`kubectl get nodes` does work, however.  Am I missing a dependency?

**Comment by robscott** [comment]
Hey @lypht, I just pushed some changes that should help. I also just noticed that you were also working on some similar Readme updates at the same time, hopefully what I wrote makes sense as well. I'll do a bit more testing, and if all looks good I'll get this out in a new release.

**Comment by robscott** [comment]
Hey @lypht, as far as I can tell this issue should be fixed with the latest release (0.1.2). If it's not, don't hesitate to open this back up. Thanks for the help!

**Comment by lypht** [comment]
That definitively fixes the issue.  Thanks!
